plugins/zlux-agent/doc/swagger/server-plugins-api.yaml

```diff
@@ -0,0 +1,784 @@
+
+  /server/log:
+    get:
+      tags:
+      - "server"
+      summary: "Retrieve the server log"
+      description: "Retrieve the server log"
+      produces:
+      - "application/json"
+      responses:
+        200:
+          description: "Success"
+        401:
+          description: "Unauthorized"
+        501:
+          description: "Log not found"
+        506:
+          description: "Set dataserviceAuthentication.rbac to true in server configuration"
+          
+  /server/logLevels:
+    get:
+      tags:
+      - "server"
+      summary: "Get the log levels of the server and plugins"
+      description: "Get the log levels of the server and plugins"
+      produces:
+      - "application/json"
+      responses:
+        200:
+          description: "Success"
+          schema:
+            $ref: "#/definitions/logLevelDefinition"
+        401:
+          description: "Unauthorized"
+        506:
+          description: "Set dataserviceAuthentication.rbac to true in server configuration"
+          
+  /server/logLevels/name/:componentName/level/:level:
+    post:
+      tags:
+      - "server"
+      summary: "Set the log 'level' using the component 'name' of a logger"
+      description: "Set the log 'level' using the component 'name' of a logger"
+      produces:
+      - "application/json"
+      parameters:
+      - in: "query"
+        name: "componentName"
+        description: "Logger component name"
+        type: "string"
+        required: true
+      - in: "query"
+        name: "level"
+        description: "Logger level (default 0 - 5)"
+        type: "number"
+        required: true
+      responses:
+        200:
+          description: "Success"
+        400:
+          description: "Log level must be a number\n\nOR\n\nLog level must be within the accepted levels of 'MIN' and 'MAX'"
+        401:
+          description: "Unauthorized"
+        506:
+          description: "Set dataserviceAuthentication.rbac to true in server configuration"
+          
+  # /server/logLevels/pattern/:componentPattern/level/:level:
+  #   post:
+  #     tags:
+  #     - "server"
+  #     summary: "Set the log levels of a logger using a RegEx 'pattern'"
+  #     description: "Set the log levels of a logger using a RegEx 'pattern'"
+  #     produces:
+  #     - "application/json"
+  #     parameters:
+  #     - in: "query"
+  #       name: "pattern"
+  #       description: "Logger component pattern"
+  #       type: "string"
+  #       required: true
+  #     responses:
+  #       200:
+  #         description: "Success"
+  #       400:
+  #         description: "Log level must be a number\n\nOR\n\nLog level must be within the accepted levels of 'MIN' and 'MAX'"
+  #       401:
+  #         description: "Unauthorized"
+  #       506:
+  #         description: "Set dataserviceAuthentication.rbac to true in server configuration"
+          
+  /server/reload:
```

Comment:
example of a route that doesnt exist on zss, therefore shouldnt be here.

Reply:
yes, separated into separate swagger for agent &server